### PR TITLE
cn script update

### PR DESCRIPTION
**Pull request by copyliu**

### CODE REVIEW

File: script/cn/code.txt [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
+	if %track_id == 160 mov $track_id,"video\sub\160_chn.ass" : return
+	if %track_id == 161 mov $track_id,"video\sub\161_chn.ass" : return
+	if %track_id == 186 mov $track_id,"video\sub\186_chn.ass" : return
+	if %track_id == 240 mov $track_id,"video\sub\240_chn.ass" : return
+	if %track_id == 241 mov $track_id,"video\sub\241_chn.ass" : return
+	if %track_id == 1014 mov $track_id,"video\sub\op4_chn.ass" : return
+	if %track_id == 1017 mov $track_id,"video\sub\op56_chn.ass" : return
+	if %track_id == 1018 mov $track_id,"video\sub\op78_chn.ass" : return
+	if %legacy_op == off && %track_id == 1012 mov $track_id,"video\sub\chn.ass" : return
 	if %legacy_op == on  && %track_id == 1012 mov $track_id,"legacy\sub\eng.ass" : return
```
Comment (vit9696) [inline script/cn/code.txt]: I think that one was missed.

Comment (copyliu) [reply script/cn/code.txt]: Let me check it later

**Comment by copyliu** [comment]
I contacted SNS team they will move their staging work to here. They were finished ep7 afaik

**Comment by copyliu** [comment]
updated

**Comment by vit9696** [comment]
Merged

**Pull request merged**